tests/net/ptp/clock/src/main.c

```diff
@@ -175,29 +175,40 @@ static u64_t timestamp_to_nsec(struct net_ptp_time *ts)
 	return (ts->second * NSEC_PER_SEC) + ts->nanosecond;
 }
 
+struct ptp_context {
```

Comment:
how does this patch relate to 64 bits support? (commit title is wrong btw, should be "tests/net: Fix PTP test case" for instance)